packs/data/equipment.db/alchemical-fire-barrel.json

```diff
@@ -0,0 +1,105 @@
+{
+    "_id": "l6fTNWsQc85AYpTm",
+    "img": "systems/pf2e/icons/equipment/consumables/other-consumables/black-powder-keg.webp",
+    "name": "Alchemical Fire Barrel",
+    "system": {
+        "activation": {
+            "condition": "",
+            "cost": 0,
+            "type": ""
+        },
+        "autoDestroy": {
+            "_deprecated": true,
+            "value": true
+        },
+        "autoUse": {
+            "_deprecated": true,
+            "value": true
+        },
+        "baseItem": null,
+        "charges": {
+            "_deprecated": true,
+            "max": 0,
+            "value": 0
+        },
+        "consumableType": {
+            "value": "ammo"
+        },
+        "consume": {
+            "_deprecated": true,
+            "value": ""
+        },
+        "containerId": null,
+        "description": {
+            "value": "<p>Ammunition for a firedrake.</p>"
```

Comment:
Are these descriptions in G&G?

Reply:
No. The way they show ammunition is in the siege weapon statblock (presumably to conserve page count). e.g. **Ammunition** alchemical fire barrel (50 gp, 3 Bulk). I can remove it since it isn't RAW. Is that the best way to handle it?

Reply:
We don't usually improvise descriptions like that, but I'll leave it up to data entry to decide.

Reply:
Well, we sort of improvised most ammunition names themselves given the vague descriptions in the book. I assume these are intended to work as part of #3174 so I have not really dug into this yet, but I will this weekend.